proposals/top-level-statements.md

```diff
@@ -183,7 +183,7 @@ static class $Program
 ### Scope of top-level local variables and local functions
 
 Even though top-level local variables and functions are "wrapped" 
-into the generated entry point method, they should still be in scope throughout the program in
+into the generated entry point method, they should still be in-scope throughout the program in
```

Comment:
Please revert. `in` is a modifier on `scope`. Correct usage of a hyphen would be when there are multiple modifiers modifying the same noun.

Reply:
Please see the following example;  
`in-house software`  
Hyphenation is used in this situation due to the adjective `in-house` ascribing a single property, that of being developed internally, to the noun `software`.  
Similarly, in the following phrase;  
`in-scope variables`  
Hyphenation is used due to the adjective `in-scope` ascribing a single property, that of being in the scope, to the noun `variables`.

Thus, I decided to use hyphenation in the context of this commit as I assume the writer intended to describe the local variables and functions to be in the scope, and, thus, to be in-scope.

Reply:
`scope` is a noun. It is the thing that the variables are in. The hyphen is not correct.